resources/guides/laptop_guide.md

```diff
@@ -0,0 +1,72 @@
+---
+id: laptop_guide
+title: Laptop Requirements Guide
+sidebar_label: Laptop Requirements Guide
+slug: /guides/laptop
+---
+
+_Last updated on 2022/08/24_
+
+Here are a few things to keep in mind when purchasing/configuring your laptop:
+
+## Hardware
+
+Although the following specifications are not strictly required, having these minimum specifications will make your computer less frustrating to use:
+
+-   8 GB of RAM
+    -   for gaming/future-proof purposes, 12 or 16 GB is more preferable
+-   256 GB of SSD storage
+    -   UWindsor provides 1 TB OneDrive cloud storage. However, if you download a lot of games and software, or if you simply prefer more local storage, you should upgrade to 512 GB / 1 TB storage
```

Comment:
I would honestly bump this up to 512GB. Even though we do get 1TB OneDrive for free, it's a massive pain to work with unless you have hot files saved to device all the time anyway. Not to mention, at this point half-TB SSDs are quite common.

Reply:
I would just give a bit more info. 256 is fine as an absolute minimum (source: my laptop has 256gb). But I would add a clause that it's strongly recommended to up to 512gb